store/transfer_processor.go

```diff
@@ -182,7 +186,22 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 		}
 		if hasTo {
 			to := common.BytesToAddress(e.To)
-			if addrDiff[contractAddr][to] == nil {
+			if bytes.Equal(e.From, model.RewardToMiner.Bytes()) && bytes.Equal(e.Address, model.ETHBytes) {
```

Comment:
the check `bytes.Equal(e.Address, model.ETHBytes)` is not necessary.
we will never write an event where from == RewardToMiner and address != ETHBytes.
if you really want to check, we should return error when address == ETHBytes. but i don't think it needed.